proprietary/tokens/src/components/ams/button.tokens.json

```diff
@@ -12,41 +12,44 @@
       "disabled": {
         "cursor": { "value": "{ams.action.disabled.cursor}" }
       },
+      "forced-color-mode": {
+        "border": { "value": "{ams.border-width.md} solid ButtonBorder" }
```

Comment:
Do we keep `ButtonBorder`?

Reply:
This uses a [system color](https://developer.mozilla.org/en-US/docs/Web/CSS/system-color) for forced colors mode, which makes sense imo.

Reply:
Agreed. But should we declare it explicitly? My understanding is that [the browser forces this](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors#usage_notes) already and “In general, web authors should not be using the forced-colors media feature to create a separate design for users with this feature enabled.”

Reply:
Without `ButtonBorder` the browser uses the currentColor I think, which is the same as the button text color. So we do not need to define it explicitly.